_posts/2021-03-01-changelog-142.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #142"
+date: 2021-03-01 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6877 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6835 -->
+* Debugging is now available in [PyCharm Professional](https://www.jetbrains.com/pycharm/) starting with 2021.1 EAPs
+
+* The [Native Debugging Support](https://plugins.jetbrains.com/plugin/12775-native-debugging-support) plugin is now
+  being suggested when you try to debug your Rust code in [PyCharm Professional](https://www.jetbrains.com/pycharm/)
+  and [GoLand](https://www.jetbrains.com/go/)
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6807 -->
+* `Change Signature` (`Refactor | Change Signature` or <kbd>Ctrl+F6</kbd>) refactoring now allows you to choose a
+  default value for new parameters (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-142/6807" w="700px" %}
+
+* Suggest installing `wasm32-unknown-unknown` target if it isn't installed yet when you try to execute `wasm-pack`
+  run configuration
+
+## Fixes
+
+* Improvements in [new name resolution](https://github.com/intellij-rust/intellij-rust/issues/6217) engine
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6734 -->
+    * Fix name resolution of macro calls inside file included by `include!` macro
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6730 -->
+    * Support file level `cfg` attributes
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6721 -->
+    * Support basic multi-resolution of inline modules
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6720 -->
+    * Fix name resolution inside cfg-disabled inline `mod`
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6773 -->
+    * Use old name resolution in test/bench non-workspace crates
+
+  Note, you can enable the new engine via `Use experimental name resolution engine` option in
+  `Preferences | Languages & Frameworks | Rust` settings
+
+* Refactoring improvements
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6874 -->
+    * Properly handle default type arguments in `Extract Function` (`Refactor | Extract Method` or <kbd>Ctrl+F6</kbd>) refactoring
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6005 -->
+    * Fix inlining of functions that reside in a different module when performing `Inline function` refactoring
```

Comment:
```suggestion
    * Delete redundant imports properly when performing `Inline function` refactoring on a function declared in a different module
```
IIUC this fix is only about deleting redundant imports, please correct if I'm wrong